### PR TITLE
Search-Task completed.

### PR DESCRIPTION
**Pull request by ranjith210297**
Hello sir, my search functionality is completed and ready to be integrated. please review it.

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -116,3 +119,31 @@ def userHome(Username):
...
+	else:
+		result = request.form.get("search")
+		result = '%'+result+'%'
+		search_result = Books.query.filter(or_(Books.tittle.ilike(result), Books.author.ilike(result), Books.isbn.ilike(result))).all()
+		return render_template("search.html", books=search_result)
+
+@app.route("/bookpage",methods=["GET"])
+def bookpage():
+	return render_template("bookpage.html")
+||||||| parent of ef5bb0e... last commit for search-resolved conflict
```
Comment (ashishk144) [inline project1/application.py:137]: why is there this code?

File: project1/application.py [hunk 2]
```diff
@@ -116,3 +119,31 @@ def userHome(Username):
     if Username in session:
         return render_template("userDetails.html", username=Username, message="Successfully logged in.", heading="Welcome back")
     return redirect(url_for('index'))
+<<<<<<< HEAD
+
+@app.route("/search",methods=["POST","GET"])
+def search():
+	if request.method == "GET":
```
Comment (ashishk144) [inline project1/application.py:126]: check whether the user is logged in.

File: project1/templates/search.html [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+    </div>
+
+
+    </tbody>
+    </table>
+
+</body>
+</html>
+||||||| parent of ef5bb0e... last commit for search-resolved conflict
```
Comment (ashishk144) [inline project1/templates/search.html:100]: seems like there are merge conflicts, resolve them before pushing it here.

File: project1/templates/search.html [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+			<div class="content">
+
+
+
+				<h2>Search for any book using ISBN, Book Name or Author name</h2>
+				<form>
+				<div>
+         			<input type="text" class="form-control" id="search" name="search" placeholder="Enter name,isbn or author">
+         			<span class="input-group-btn">
+         			<!-- <button class="btn" type="submit" formmethod="POST" formaction="{{ url_for('search') }}">Search</button> -->
```
Comment (ashishk144) [inline project1/templates/search.html:50]: remove commented code

File: project1/templates/search.html [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+</body>
+</html>
+||||||| parent of ef5bb0e... last commit for search-resolved conflict
+=======
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" type="text/css" href="D:\MSIT\GIT_REPOS\project-1-2019501102\project1\static/search_style.css">
```
Comment (ashishk144) [inline project1/templates/search.html:107]: Use generic layout and extend these files.

File: project1/application.py [hunk 6]
```diff
@@ -116,3 +119,31 @@ def userHome(Username):
...
+=======
+
+@app.route("/search",methods=["POST","GET"])
+def search():
+	if request.method == "GET":
+		return render_template("search.html")
+	else:
+		result = request.form.get("search")
+		result = '%'+result+'%'
+		search_result = Books.query.filter(or_(Books.tittle.ilike(result), Books.author.ilike(result), Books.isbn.ilike(result))).all()
```
Comment (ashishk144) [inline project1/application.py:147]: what if there are no search results, how are you handling that?

**Review by ashishk144 — COMMENTED** [review]
Please remove all the extra bits and pieces of code. Use the generic template and extend the html files to it. Work on rest of the feedback provided.

File: project1/application.py (hunk 1 above)
Comment (ranjith210297) [reply project1/application.py:137]: actually it was a conflict, resolved now.

File: project1/templates/search.html (hunk 3 above)
Comment (ranjith210297) [reply project1/templates/search.html:100]: ya resolved.

File: project1/templates/search.html (hunk 4 above)
Comment (ranjith210297) [reply project1/templates/search.html:50]: Completed.

File: project1/application.py (hunk 6 above)
Comment (ranjith210297) [reply project1/application.py:147]: ya im going to add some code to do that, will do that while  integration.

File: project1/application.py (hunk 2 above)
Comment (ranjith210297) [reply project1/application.py:126]: ues, i added that functionality.

**Pull request closed without merging**